source/updating.rst

```diff
@@ -0,0 +1,42 @@
+Tutorial: Updating Fetch Software
+===========================
+
+Updating your Fetch Research Edition Robot software in the proper manner is
+important to keep your Robot functioning smoothly. In addition you will get
+the latest capabilities and bug fixes added by Fetch Robotics. Fetch Robotics
+recommends updating robot software on a regular basis.
+
+Understanding Software Updates
+-------------------------
+
+Fetch Research Edition packages will be updated with new capabilities and bug
+fixes as needs become apparent.
+
+Setting up apt-get Sources
+--------------------
+
+The following commands will create files so that apt on the robot can see the
+public ros package server as well as the Fetch Robotics package server. You
+should only need to do this once, unless someone alters or deletes them.
```

Comment:
Did I tell you that VT chuong guy emailed me after the fact to thank me for fixing the robot he bricked? He said he was blocked from replying until then. haha